GOVERNANCE.md

```diff
@@ -1,43 +1,102 @@
+* They respond to PR review requests in a timely manner. Generally, a response is expected within 24 hours of the PR being submitted.
+* They ensure that code changes they approve:
+  * Meet the [coding conventions](/CONTRIBUTING.md) required by the Project. This includes ensuring the code is sufficiently well tested, follows the appropriate standards, and, of course - does not break the build.  
+  * Is consistent with the goals and direction of the Project. This requires not just code and architectural correctness, but also ensuring that it does not introduce "scope creep," and does not unduly affect existing users.
+* Once a PR has the requisite approvals, the last approving maintainer is responsible for merging the change (however, the PR's author must ensure the change is merge-ready).
+
+Maintainers are documented in each repository's [CODEOWNERS](https://help.github.com/articles/about-codeowners/) file. To become a maintainer, one must:
+
+* Work in a helpful and collaborative way with the community
+* Have a track record of providing constructive feedback on others' PRs
+* Have submitted at least 20 PRs themselves
+
+The process for nominating and approving Maintainers is:
+
+* Open a PR against the CODEOWNERS file that covers the parts of the project you wish to nominate someone (or yourself) for
+* A consensus of existing Maintainers must approve your PR
+
+### The in-toto Steering Committee (ITSC)
+
+The in-toto Project is governed by the [ITSC](https://github.com/in-toto/community/blob/main/ITSC.md) that is exclusively responsible for in-toto's [specification](https://github.com/in-toto/docs), [enhancements](https://github.com/in-toto/in-toto/ite), and the Project's strategic goals and direction. ITSC members have final authority over:
+
+* Technical direction of the Project and subprojects
+* Project governance and process (this document).
+* Contribution policy
+* Conflict resolution
+
+The ITSC adheres to the following:
+
+* Be comprised of at least five (5) members with representation from both Academia and Industry.
+* No more than 2 ITSC members may be affiliated with the same organization.
+* At least 40% of the ITSC must be represented by organizations that currently have an in-toto implementation deployed in production OR that are actively working to integrate in-toto into other software supply chain efforts or standards.
+* Each ITSC member's term is 24 months.
+* There is no limit to the number of terms an ITSC member can serve.
+* ITSC members may remove themselves voluntarily at any time.
+
+For more information about the ITSC, please refer to the ITSC Charter
+
+## Decision Making
+
+Maintainer and ITSC decisions are made by a [lazy consensus](http://rave.apache.org/docs/governance/lazyConsensus.html) approach. When formal voting is required, members may abstain. Negative votes must be accompanied by an explanation or alternative proposal.
```

Comment:
Should this detail where the votes take place? in-toto-public to allow it to be asynchronous?